### DISCUSSION TITLE
Is there a way to set keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by usmanpm** [MonitorControl/MonitorControl#409] [Q&A] [answered]
First of all, a huge thank you for this utility (and saving my eyes)

I was wondering if there's a way to set custom keyboard shortcuts? I use an Apple Keyboard and "BetterTouchTool" for assigning keyboard shortcuts to the extended keys. I was wondering if I could somehow have F1/F2 control the MacBook's brightness, and F5/F6 control the external display's brightness?

**Comment by JoniVR** [comment, marked as answer]
See: #118, haven't had the time for it yet, but we're always open to Pull Requests if people want it bad enough.

**Reply by usmanpm** [reply]
No worries, I found a hacky way of doing it using BetterTouchTool. Thanks!
